Blogpost call gruppentreffen (#217)
DE-RSE/de-rse.github.io: _posts/2021-06-01-rse-gruppentreffen.md

---
title: "1. Treffen von RSE-Gruppen in Deutschland"
layout: post
author: "Frank Löffler"
menulang: de
---

> An wissenschaftlichen Instituten und Universitäten in Deutschland entstehen immer mehr RSE-Gruppen.

Reviewer: ```suggestion
An Forschungsinstituten und Universitäten in Deutschland entstehen immer mehr RSE-Gruppen.
```
(Was sind wissenschaftliche Institute?)

Weil wir aber glauben, dass wir gemeinsam stärker sind und alle von einem Ideen- und Erfahrungsaustausch profitieren können, soll es ein 1. (virtuelles, halbtägiges) Treffen von RSE-Gruppen noch im Juni geben.
Wir wollen damit vor allem die Leitung der Gruppen ansprechen, aber auch alle anderen, die sich dafür interessieren (vielleicht selbst in die Richtung gehen wollen).
Dazu gibt es eine [Terminumfrage](https://terminplaner4.dfn.de/mowXBWKf18FJDZaa) (auszufüllen bis 2021-06-06).
Angedacht sind später regelmäßige Treffen alle paar Monate, je nach dem, wie wir selbst Bedarf sehen.

Im bisherigen Orga-Team (wir sind offen für jede/n) haben wir auch schon über mögliche Themen der ersten Treffen diskutiert und sind bei folgendem Plan gelandet:

1. Treffen (Juni 2021)
    - RSE-Gruppenaufbau/leitung/vernetzung/infrastruktur
    - Selbstvorstellung (der Gruppe)
        - 1-3 Minuten, mit verlinktem Material
    - viel Vernetzungszeit / offene Diskussionszeit / Breakout-Räume
2. Treffen
    - Forschungsdatenmanagement, speziell elektronische Laborbücher (ELNs)
3. Treffen
    - RSEs in der Lehre
    - RSE-Kommunikation
    - Outreach/PR
4. Treffen
    - Förderung von RSEs / Software

> Dabei sind die Themen nicht in Stein gemeißelt, nur die Vernetzung wird beim 1. Treffen einfach Hauptrolle spielen müssen.

Reviewer: ```suggestion
Dabei sind die Themen nicht in Stein gemeißelt, lediglich die Vernetzung sollte beim 1. Treffen eine Hauptrolle spielen.
```

Macht für mich so mehr Sinn (wenn richtig interpretiert).

> Wen interessiert, wie wir zu diesen Themen bekommen sind, kann sich das [hier](https://github.com/DE-RSE/projekte/issues/3) ansehen bzw. dort sogar noch neue Themen anbringen oder bestehende kommentieren.

Reviewer: ```suggestion
Wen interessiert, wie wir zu diesen Themen gekommen sind, kann sich das [hier](https://github.com/DE-RSE/projekte/issues/3) ansehen bzw. dort sogar noch neue Themen einbringen oder bestehende kommentieren.
```

Typo + Wording.
